samples/boards/nrf/clock_skew/src/main.c

```diff
@@ -198,15 +198,14 @@ static void sync_work_handler(struct k_work *work)
 
 void main(void)
 {
-	const char *clock_label = DT_LABEL(CLOCK_NODE);
-	const char *timer0_label = DT_LABEL(TIMER_NODE);
 	uint32_t top;
 	int rc;
 
 	/* Grab the clock driver */
-	clock0 = device_get_binding(clock_label);
-	if (clock0 == NULL) {
-		printk("Failed to fetch clock %s\n", clock_label);
+	clock0 = DEVICE_DT_GET(CLOCK_NODE);
```

Comment:
Why not initialize this pointer (and `timer0` as well) statically?

Reply:
no reason, will make the change.